community-operators/okd-pipeline-operator/1.2.3/manifests/openshift-pipelines-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,363 @@
+--- 
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata: 
+  annotations: 
+    support: OKD working group
+    alm-examples: |-
+        [
+          {
+            "apiVersion": "operator.tekton.dev/v1alpha1",
+            "kind": "Config",
+            "metadata": {
+              "name": "name.must.be-cluster"
+            },
+            "spec": {
+              "targetNamespace": "openshift-pipelines"
+            }
+          }
+        ]
+    capabilities: "Seamless Upgrades"
+    containerImage: quay.io/jomeier/tektoncd-pipeline-operator
+    description: >-
+        OKD Pipelines is a cloud-native CI/CD solution for building
+        pipelines using Tekton concepts which run natively on OpenShift, Kubernetes and OKD.
+    createdAt: '2021-01-31T18:16:25Z'
+    repository: 'https://github.com/jomeier/tektoncd-pipeline-operator/tree/okd-v1.2.3'
```

Comment:
Hi @jomeier !
This is based off a deprecated project https://github.com/openshift/tektoncd-pipeline-operator .